apps/site/tests/e2e/general-behavior.spec.ts

```diff
@@ -0,0 +1,225 @@
+import { test, expect } from '@playwright/test';
+
+const selectors = {
```

Comment:
I am personally a fan of [test IDs](https://playwright.dev/docs/locators#locate-by-test-id), would we maybe consider using them? :slightly_smiling_face:

Reply:
My only concern is that they will be leftover in production, right?

Reply:
Yes, but I think those are pretty common and harmless, even here on github on this PR, if you open the console and type `document.querySelectorAll('[data-testid]')` you'll see a bunch of elements being returned (or also on https://nextjs.org/)

So I personally don't think that they remaining in production is a problem 🤔